### PR TITLE
Hero block: enable mobile, tablet, desktop sizes for video backgrounds

### LINKED ISSUE #157: Hero block: enable mobile, tablet, desktop sizes for video backgrounds
Currently only the first video link will render as the background for the hero block, for all sizes.
The rest will just be a link.
image

The live site has mobile, tablet AND desktop mp4s, however perhaps we can ask the customer if they can simplify and just have mobile + desktop (like the blog article hero images)?

image

ALSO -- notice the CTA button for that hero banner shows up under the right header nav when scrolling down.
image

### PR DESCRIPTION
**Pull request by RitwikSrivastava**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Add support for multiple video as per screen width
Fix #157 

Test URLs:
- Before: https://main--sling--aemsites.aem.live/
- After: https://herovideo--sling--aemsites.aem.live/

### CODE REVIEW

File: aemedge/blocks/hero/hero.js [hunk 1]
```diff
@@ -1,6 +1,25 @@
 // import { createOptimizedPicture } from '../../scripts/aem.js';
 import { createTag } from '../../scripts/utils.js';
 
+function getVideoUrl(videoLinks) {
+  const screenWidth = window.innerWidth;
+
+  if (screenWidth >= 1024) {
+    // Desktop view (1024px and above)
+    return videoLinks.find((link) => link.getAttribute('href').includes('desktop'))?.getAttribute('href');
```
Comment (helms-charity) [inline aemedge/blocks/hero/hero.js:9]: It was a good idea, but I think the naming convention is not standardized. You will have to select the video based on the order. 

The blog pages have the desktop image 1st, and mobile image last, so let's use the same order for the videos. 
- 1st video link is destop, 
- 2nd is tablet, and 
- 3rd is mobile. 
- If there are only 2 videos, the use the 1st video for both desktop and tablet, and the 2nd one for mobile.

Current sling.com homepage video names for example:
image

**Review by helms-charity — CHANGES_REQUESTED** [review]
see comment

**Review by helms-charity — APPROVED** [review]
Looks great!

**Comment by RitwikSrivastava** [comment]
Added in the SideKick Library too.

**Pull request merged**